autorest/codegen/templates/client.py.jinja2

```diff
@@ -11,6 +11,9 @@
 
     {{ op_tools.serialize_with_wrap(serializer.property_descriptions(async_mode), "\n ") | indent }}
     {{ serializer.init_signature_and_response_type_annotation(async_mode) | indent }}
+    {% if serializer.should_init_super %}
```

Comment:
why do we not init super for abstract? it shouldn't super matter

Reply:
The `should_init_super` is True only when it's mixin and there is abstract operation.
I added `super().__init__()` for this case so that it can have a chance to check abstract implementation. I think we don't need `super().__init__()` in other cases?